jerry-core/debugger/debugger.c

```diff
@@ -514,6 +519,10 @@ jerry_debugger_breakpoint_hit (uint8_t message_type) /**< message type */
 
   while (!jerry_debugger_receive ())
   {
+    /* timeout */
+    #ifdef __unix__
+    syscall(SYS_nanosleep, (const struct timespec[]){{0, 100}}, NULL);
```

Comment:
But if the jerry libc is turned of then we are using the standard libc which should have the nanosleep function.